lib/rester/client.rb

```diff
@@ -33,13 +37,17 @@ def connected?
       adapter.connected? && adapter.get('/ping').first == 200
     end
 
+    def circuit_breaker_enabled?
+      !!@_breaker_enabled
+    end
+
     def request(verb, path, params={})
-      @_request_breaker.call(verb, path, params)
-    rescue Utils::CircuitBreaker::CircuitOpenError
-      # Translate this error so it's easier handle for clients.
-      # Also, at some point we may want to extract CircuitBreaker into its own
-      # gem, and this will make that easier.
-      raise Errors::CircuitOpenError
+        @_requester.call(verb, path, params)
+      rescue Utils::CircuitBreaker::CircuitOpenError
+        # Translate this error so it's easier handle for clients.
+        # Also, at some point we may want to extract CircuitBreaker into its own
+        # gem, and this will make that easier.
+        raise Errors::CircuitOpenError
```

Comment:
These are tabbed over one too many.

This can be:
```ruby
def request(verb, path, params={})
  @_requester.call(verb, path, params)
rescue Utils::CircuitBreaker::CircuitOpenError
  ...
end
```